media/static-bundles.json

```diff
@@ -1065,6 +1071,17 @@
       ],
       "name": "firefox_whatsnew_65_newsletter"
     },
+    {
+      "files": [
+        "js/libs/jquery.waypoints.min.js",
```

Comment:
Is this file needed? (you seem to be using `position: sticky` which is wayyyy more performant 💯)

Reply:
good catch! Forgotten file. 🙂